StickyHeaders/src/main/java/io/doist/recyclerviewext/sticky_headers/StickyHeadersLinearLayoutManager.java

```diff
@@ -83,6 +89,30 @@ private void setAdapter(RecyclerView.Adapter adapter) {
     }
 
     @Override
+    public Parcelable onSaveInstanceState() {
+        Parcelable superState = super.onSaveInstanceState();
+
+        SavedState ss = new SavedState();
+        ss.superState = superState;
```

Comment:
To make this shorter, we could have `ss.superState = super.onSaveInstanceState();` instead of a variable. Either way is fine though.